
package mx.com.profuturo_gnp.ws.sso.controlacceso;

import javax.xml.ws.WebFault;


/**
 * This class was generated by the JAX-WS RI.
 * JAX-WS RI 2.1.7-b01-
 * Generated source version: 2.1
 * 
 */
@WebFault(name = "excepcionMsgFault", targetNamespace = "http://www.profuturo-gnp.com.mx/ws/sso/controlacceso")
public class SSOFault
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private ExcepcionSSO faultInfo;

    /**
     * 
     * @param message
     * @param faultInfo
     */
    public SSOFault(String message, ExcepcionSSO faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param message
     * @param faultInfo
     * @param cause
     */
    public SSOFault(String message, ExcepcionSSO faultInfo, Throwable cause) {
        super(message, cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: mx.com.profuturo_gnp.ws.sso.controlacceso.ExcepcionSSO
     */
    public ExcepcionSSO getFaultInfo() {
        return faultInfo;
    }

}
